servicetalk-router-api/src/main/java/io/servicetalk/router/api/NoOffloadsRouteExecutionStrategy.java

```diff
@@ -27,8 +25,8 @@
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * A special {@link RouteExecutionStrategy} that indicates that a resource class or method needs to use
- * the {@link HttpExecutionStrategies#noOffloadsStrategy()}.
+ * A special {@link RouteExecutionStrategy} that indicates that a resource class or method needs to use a strategy that
+ * does not offload user code to a worker thread pool.
```

Comment:
nit: remove usage of word "worker" => `does not offload user code using a different thread pool`

(here and below)